docs/api.rst

```diff
@@ -227,3 +227,87 @@ Run a single single unit test one can:
 To get test results with coverage report:
 
     py.test  --cov=plugin_package
+
+Feature Deprecation
+===================
+
+Parsers and other parts of the framework go through periodic revisions and
+updates, and sometimes previously used features will be deprecated.  This is
+a three step process:
+
+1. The outdated function, method or class is marked as pending deprecation.
+   Code using this feature now generates a warning when the tests are run,
+   but otherwise works.  At this stage anyone receiving a warning about
+   pending deprecation SHOULD change over to using the new functionality or
+   at least not using the deprecated version.
+2. Once sufficient time has elapsed, the outdated feature is marked as
+   deprecated.  The py.test tests will fail with a fatal error, and any code
+   checked in that uses deprecated features will not be able to be merged
+   because of the tests failing.  Anyone receiving a warning about pending
+   deprecation MUST fix their code so that it no longer warns of deprecation.
+3. Once sufficient time has elapsed, the outdated feature is removed from the
+   code.  Any code still using this feature will fail its tests because the
+   feature it requires is absent.
+
+The usual time between each step should be two minor versions of the Insights
+core.
+
+To deprecate code, call the :py:func:``insights.util.deprecated`` function from
```

Comment:
@PaulWay I believe you need to use single back ticks here to get Sphinx to interpret it as a link.